src/openrct2-ui/scripting/ScViewport.hpp

```diff
@@ -275,7 +275,7 @@ namespace OpenRCT2::Scripting
                 {
                     viewport->viewPos.x = left;
                     viewport->viewPos.y = top;
-                    viewport->flags &= ~WF_SCROLLING_TO_LOCATION;
+                    w->flags &= ~WF_SCROLLING_TO_LOCATION;
```

Comment:
That is interesting, pretty sure without setting the flag on the window this has never worked before. Should we add another changelog entry for that?

Reply:
It removes the flag from the window, rather than set it, though. Which is probably less likely to result in visible bugs.

I’m fine with putting it in the changelog, but I’m not sure how this bug would manifest itself, and thus how to summarise it.

Reply:
Good point, I think if the scroll to location was used and then immediately sets the viewport location it would probably keep scrolling but I also have no idea if anyone ever managed to get to this point, so probably not worth it.